src/clj/athens/self_hosted/components/datahike.clj

```diff
@@ -57,7 +57,7 @@
     :db/valueType   :db.type/boolean
     :db/cardinality :db.cardinality/one}
    {:db/ident       :page/sidebar
-    :db/valueType   :db.type/long
+    :db/valueType   :db.type/bigint
```

Comment:
Is this really necessary?
What is the error you're getting because of `:db.type/long`?

Reply:
Updated. Reverted to `:db.type/long`.

Reply:
Previously, I was getting an error when the client is remote because the resolver for `:datascript/page-add-shortcut` is returning an `int`. One solution is to coerce it to `long` but it won't work in cljs that's why I instead changed this to `long`. Another option I considered is to use a reader conditional.

Anyway, in the latest update, I reverted this back to `:db.type/long` since there's no more issue with the `:db/valueType`.

Reply:
Updated again. This is now `:db.type/number`